src/app/task/task-list/task-list.component.ts

```diff
@@ -16,7 +17,7 @@ export class TaskListComponent implements OnInit, OnDestroy {
   private destroy$: Subject<void> = new Subject<void>();
   taskList : Task[] = new Array();
 
-  constructor(private taskService : TaskService) {}
+  constructor(private taskService : TaskService, private authService: AuthService) {}
 
   ngOnInit(): void {
     this.getTasks();
```

Comment:
additionally what you could is instead of using just a simple taskList Task[] array you can use an observable and then using async pipe display data on the template, wouldn't have to subscribe to everything and it would be easier for every method